src/components/Sum.jsx

```diff
@@ -1 +1,7 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
```

Comment:
Sum.defaultProps = {
  a: 0,
  b: 0,
}

I think we shoul use this for default in this task

Reply:
I believe we can destructure the props of Sum with default values without defaultProps since our default values are short enough as well as the properties themselves.

Reply:
let's add `default props` and `prop types `to learn how they work